src/components/Products.js

```diff
@@ -0,0 +1,181 @@
+                            {categoryData && categoryData.length ? categoryData.filter((item) => { return item.order >= 0 }).map((data, i) => (<option value={data.id} className={this.state.catId ? data.id == this.state.catId ? 'selected' : '' : ''} key={i}>
+                                {data.name}
+                            </option>)) : null}                            
+                        </select>
+                    </div>
+                </section>
+                <section className="col span-9-of-12 product-list-area">
+                    <div className="prod-list">
+                        <ul className="rest-tiles">
+                        {productList && productList.length ? productList.filter((data)=>{return this.state.catId ? this.state.catId == data.category : data }).map((item,i)=>(<li key={i}>
+                            <p className="prod-name" title={item.name}>{item.name}</p>
+                            <div className="prod-detls">
+                            <div className="prod-img">
+                                <figure>
+                            <LazyLoad height={75} offset={500} once>
+                                <img className={`rocketImg`}
+                                    alt={item.name}
+                                    src={window.location.origin + item.imageURL}
+                                />
+                            </LazyLoad>
+                            </figure>
+                            </div>
+                            <div className="prod-infodetails">
+                            <div className="description-area">
+                                <article className="prod-desc truncate-overflow" title={item.description}>{item.description}</article>
+                            </div>
+                            <div className="buy-area">
+                                <div className="col span-1-of-2 price">
+                                   <p> MRP Rs.{item.price}</p>
+                                </div>
+                                <div className="col span-1-of-2 buy-btn">
+                                    <button className="btn" onClick={(event)=>this.addToCart(event,item)} aria-label={`Click on this button to buy ${item.name} for ${item.price} rupees.`}>Buy Now</button>
+                                    <button className="btn small-screen-btn" onClick={(event)=>this.addToCart(event,item)} aria-label={`Click on this button to buy ${item.name} for ${item.price} rupees.`}>Buy Now @ Rs.{item.price}</button>
+                                </div>
+                            </div>
+                            </div>
+                            </div>
+                        </li>)) : null}
+                        </ul>
+                    </div>
+                </section>
+            </div>
+            <Footer></Footer>
+            {openModal ? (<Modal 
+            open={openModal} 
```

Comment:
seems like you have added Modal on every page. What if your website had 100 pages, would you add Modal to each one of them? Can you think of a better solution?